src/FeatureFlag.php

```diff
@@ -16,6 +18,16 @@ public static function isEnabled($code, $context)
         return Injector::inst()->get(FeatureFlagChecker::class)->isEnabled($code, $context);
     }
 
+    public static function enable($code, $context)
```

Comment:
Can we have a comment explaining who should use this and when? Clarify, for example, that it will persistently enable the feature flag and not simply do it for this pageview.